### PR TITLE
Most popular container

### PR DESCRIPTION
**Pull request by blongden73**
## What does this change?
Changes the way that numbers in the most popular container are displayed and added to the page. 

@NataliaLKB @sndrs @GHaberis

### CODE REVIEW

File: common/app/views/fragments/collections/popular.scala.html [hunk 1]
```diff
@@ -35,7 +35,14 @@
                     @section.trails.zipWithRowInfo.map{ case (trail, info) =>
                     <li class="headline-list__item headline-column__item headline-column--tablet__item headline-column--desktop__item @toneClass(trail)--most-popular">
                         <div class="headline-list__link" data-link-name="@info.rowNum | text">
-                            <p class="headline-list__count">@info.rowNum</p>
+                            @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
+                                <p class="headline-list__count">@info.rowNum</p>
+                            }
+                            @if(experiments.ActiveExperiments.isParticipating(experiments.Garnett)){
+                                <span class="most-popular__number-@info.rowNum">
+                                    @fragments.inlineSvg(s"number-${info.rowNum}", "numbers", List("most-popular__number-svg svg-number"))
```
Comment (NataliaLKB) [inline common/app/views/fragments/collections/popular.scala.html]: Why add these classes, if you are not using them?`most-popular__number-svg svg-number`

File: static/src/stylesheets/module/_headline-list-garnett.scss [hunk 2]
```diff
@@ -58,7 +58,18 @@
...
-            margin-left: gs-span(1) - $gs-gutter / 2;
+            margin-left: gs-span(1) + $gs-gutter / 2;
+        }
+    }
+
+    .inline-numbers {
+        float: left;
+        margin-top: $gs-baseline / 4;
+
+        svg {
```
Comment (sndrs) [inline static/src/stylesheets/module/_headline-list-garnett.scss]: `fill: $garnett-neutral-1;` can go on `.inline-numbers` i think?

Comment (NataliaLKB) [inline static/src/stylesheets/module/_headline-list-garnett.scss]: Minor: You could use the `inline-numbers__svg` instead I think

Comment (blongden73) [reply static/src/stylesheets/module/_headline-list-garnett.scss]: This is fixed by removing the svg entirely.

**Review by NataliaLKB — APPROVED** [review]
Great!

**Review by sndrs — APPROVED** [review]

**Pull request merged**